### PR TITLE
Migrate layout-full to CSS grid

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Added `u-layout-grid_breakout` class for blocks that bleed out of the CSS grid to the browser window edge.

## Changes

- Add block around /consumer-tools/retirement/before-you-claim/about/ content area to fix problem where content bottom was erroneously flush.

## How to test this PR

1. These pages use the layout full template. They should not be broken:

Form explainer:
http://localhost:8000/owning-a-home/loan-estimate/

CCDB
http://localhost:8000/data-research/consumer-complaints/search/

Gov-delivery
https://www.consumerfinance.gov/govdelivery-subscribe/success/
https://www.consumerfinance.gov/govdelivery-subscribe/error/

Know Before You Owe
http://localhost:8000/know-before-you-owe/
http://localhost:8000/know-before-you-owe/compare/
http://localhost:8000/know-before-you-owe/timeline/

Rate Explorer
http://localhost:8000/owning-a-home/explore-rates/

Transcript
http://localhost:8000/transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/

Mega Menu Preview
Go into Wagtail admin and go to the Mega Menu settings and hit "Preview"

Prepaid agreements
http://localhost:8000/data-research/prepaid-accounts/search-agreements/

iRegs
http://localhost:8000/rules-policy/regulations/1002/

Before You Claim
http://localhost:8000/consumer-tools/retirement/before-you-claim/

TDP activities search
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/

Story page
http://localhost:8000/about-us/racial-equity/
http://localhost:8000/about-us/small-business-lending/
http://localhost:8000/es/quienes-somos/prestamos-pequenos-negocios/

Recent updates
http://localhost:8000/activity-log

Spanish homepage
http://localhost:8000/es/

Homepage
http://localhost:8000/

400 and 500 error pages

> 1. Temporarily change [this line](https://github.com/cfpb/consumerfinance.gov/blob/ef8522675c59e74f4d16ed86b0bc07b80bbcabec/cfgov/cfgov/settings/local.py#L18) to `False`.
> 2. Run `ALLOW_ADMIN_URL=1 cfgov/manage.py runserver --insecure`
> 3. To test the 404 template: visit http://localhost:8000/something-that-doesnt-exist/
> 4. To test the 500 template: visit http://localhost:8000/admin/failwhale/

Before you claim - Added block to content are to fix issue where there was no padding at the bottom of the content area.
http://localhost:8000/consumer-tools/retirement/before-you-claim/about/

### CODE REVIEW

File: cfgov/jinja2/owning-a-home/explore-rates/index.html [hunk 1]
```diff
@@ -28,568 +28,565 @@
 {% endblock %}
 
 {% block body_content %}
-    <div class="content_wrapper">
-        <div class="content_main u-pb0">
```
Comment (contolini) [inline cfgov/jinja2/owning-a-home/explore-rates/index.html:32]: The [viz checker](https://github.com/cfpb/cfgov-visual-regression) is showing that removing `content_main` is causing paragraphs to not get wrapped on the rate checker:

| before | after |
|--------|-------|
| image | image |

**Comment by contolini** [comment]
Also it looks like the [iregs search](https://www.consumerfinance.gov/rules-policy/regulations/search-regulations/) loses some padding:

| before | after |
|--------|-------|
| image | image |

File: cfgov/jinja2/know-before-you-owe/index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h1 class="m-hero_heading">Know Before You Owe | Mortgages</h1>
 {% endblock %}
 
 {% block body_content %}
-<div class="wrapper wrapper__match-content">
```
Comment (contolini) [inline cfgov/jinja2/know-before-you-owe/index.html:68]: There are a couple [kbyo](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/know-before-you-owe/timeline/index.html) [sub-templates](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/know-before-you-owe/compare/index.html) that probably also need updating?

**Review by contolini — APPROVED** [review]
Nice job! The only other thing the tool picked up is that there's leftover gutter padding from `content_main` on the [financial literacy pages](https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/teach/activities/):

| before | after |
|--------|-------|
| image | Screenshot 2024-03-05 at 4 49 12 PM |

**Comment by anselmbradford** [comment]
@contolini updates are in 

rate explorer/kbyo
https://github.com/cfpb/consumerfinance.gov/pull/8164/commits/b59c1691524a7578c5f62b0a53a3c5ab4e74bdcb

iregs search
https://github.com/cfpb/consumerfinance.gov/pull/8164/commits/9f8ddf2795c41b16e8d4245ac04a714dd574bee9

TDP
https://github.com/cfpb/consumerfinance.gov/pull/8164/commits/b714e22bde0dd29822fad24bafef08e432a0de25

ty ty ty!!

**Comment by anselmbradford** [comment]
FYI I rewrote the rate explorer brand footer in flexbox to reduce the styles. One consequence of this is the items in the footer wrap a little different across screen sizes. I think it's fine.

**Pull request merged**